data/questionnaires/barvo.yml

```diff
@@ -0,0 +1,27 @@
+---
+bravo:
+  1-2-2-2:
+    required_percentage_correct: 100
+    questions:
+      like_stuff:
+        multi_select: false
+        label: Do you like stuff
+        assessment_summary: Hey! You like stuff
+        assessment_fail_summary: You should like stuff!
+        answers:
+          'yes': 'Yes'
+          'no': 'No'
+        correct_answers: 
+          - 'yes'
+  1-2-2-3:
+    questions:
+      who_likes_stuff:
+        multi_select: true
+        label: Who likes stuff
```

Comment:
We also want to have demo questions we can answer. Can you replace this `stuff` with a question that anyone can correctly answer. This makes it easier to read and test.

Reply:
Not sure but probably best to add the answer to the question body then as will be a struggle to find easy random answerable questions we have quite a few questions in other modules

Reply:
Yes exactly, so like:
`Select even numbers: 1,2,3,4`
or
`Select from following:`
- correct_answer_1
- wrong_answer_1
- correct_answer_2
- wrong_answer_2

If you can update both the demo summative questionnaires and confidence check questions in such a manner

Reply:
Unless @agoldstone93 was doing it in his branch?

Reply:
@mo-zag file name is still wrong `barvo`

Reply:
I hadn't planned to, as things stand. But i'm happy to if necessary

Reply:
@mo-zag I think we can improve the spelling grammar and wording but I can tackle that as I refactor